src/components/Header/PngBalanceContent.tsx

```diff
@@ -1,10 +1,9 @@
-import { TokenAmount, JSBI, ChainId } from '@pangolindex/sdk'
+import { TokenAmount, JSBI, ChainId, CHAINS } from '@pangolindex/sdk'
 import React, { useMemo, useState } from 'react'
 import { X } from 'react-feather'
 import styled from 'styled-components'
 import tokenLogo from '../../assets/images/token-logo.png'
 import { injected } from '../../connectors'
-import { CHAINS } from '../../constants/chains'
```

Comment:
we need to remove `CHAINS` from `src/constants/chains.ts` as we dont need it now

Reply:
Updated by @jb1011 white_check_mark